onadata/apps/viewer/tasks.py

```diff
@@ -127,12 +127,10 @@ def _create_export(xform, export_type, options):
 @app.task(track_started=True)
 def create_xls_export(username, id_string, export_id, **options):
```

Comment:
Should we rename the function ? Or better yet add a new function and deprecate this one ?